components/lib/datatable/datatable.d.ts

```diff
@@ -130,9 +130,9 @@ interface DataTableDataSelectableParams {
     index: number;
 }
 
-interface DataTableSelectionChangeParams {
+interface DataTableSelectionChangeParams<TValue extends DataTableValueArray> {
     originalEvent: React.SyntheticEvent;
-    value: any;
+    value: DataTableSelection<TValue>;
     type?: string;
     [key: string]: any;
```

Comment:
`key: string` should be `TValue[keyof TValue]` no?

Reply:
Actually I am not sure what that is. All the values of rows are in `value`. `[key: string]: any` seems to not exist in the object from what I have seen but I am not sure if there is a case where it shows up.